data/hai/hai outfits.txt

```diff
@@ -680,6 +680,15 @@ outfit "Pulse Rifle"
 	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to more effectively incapacitate its target, if several bolts land in quick succession, the accumulated charge has a lethal effect on most species' nervous systems."
 
 
+outfit "Blade Of Might"
```

Comment:
Love the sword. It would be a good symbol of power for the leader of the Unfettered. Perhaps it should be mentioned in the Hai Reveal when the player meets the Unfettered leader? He couldn't use it in the duel (out of fairness) but it could be on display somewhere.

Reply:
My thinking exactly, I mentionned that earlier, but because is it isnt always online so you wouldnt notice it that easily so idk

Reply:
Would need that well written into it, you know? The thing is the warlord never considers fighting so... Mb he could bring his hand to it and then withdraw it and you notice the handler